python/src/synapse/ml/dl/DeepVisionClassifier.py

```diff
@@ -0,0 +1,181 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in project root for information.
+
+from pyspark import keyword_only
+from pyspark.ml.param.shared import Param, Params
+from synapse.ml.dl.LightningEstimator import LightningEstimator
+from synapse.ml.dl.LitDeepVisionModel import LitDeepVisionModel
+
+
+class DeepVisionClassifier(LightningEstimator):
+
+    backbone = Param(
+        Params._dummy(), "backbone", "backbone of the deep vision classifier"
+    )
+
+    num_layers_to_fine_tune = Param(
+        Params._dummy(),
+        "num_layers_to_fine_tune",
+        "number of last layers to fine tune for the model, should be smaller or equal to 3",
+    )
+
+    num_classes = Param(Params._dummy(), "num_classes", "number of target classes")
```

Comment:
Do these align with sparkML names? Not saying they dont right now, but just a good thing to double check

Reply:
You're right, they use camelCase

Reply:
But there're conflicts of this format as horovod uses snake_case... And we're extending their parameters, so should we just keep snake_case?